### PR TITLE
[Mobile] Implement BasicCalculator.

### PR DESCRIPTION
**Pull request by AchoWang**

### CODE REVIEW

**Review by wb0716 — APPROVED** [review]
Good for me

File: BasicCalculator/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# BasicCalculator
+BasicCalculator is a sample application which demonstrates how to create a calculator
+with basic mathematical operations using Xamarin.Forms library.
```
Comment (zg2nets) [inline BasicCalculator/README.md]: using Xamarin.Forms library  -> NUI Xaml

File: BasicCalculator/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* Preview of the result
+* Basic mathematical operations
+
+### Prerequisites
+
+* [Visual Studio](https://www.visualstudio.com/) - Buildtool, IDE
+* [Visual Studio Tools for Tizen](https://docs.tizen.org/application/vstools/install) - Visual Studio plugin for Tizen .NET application development
+
+### Author
+* Bartłomiej Uliasz
```
Comment (zg2nets) [inline BasicCalculator/README.md:17]: -> wang guowei

File: BasicCalculator/README.md (hunk 1 above)
Comment (AchoWang) [reply BasicCalculator/README.md]: Done

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by xuelian-bai — APPROVED** [review]
looks good to me

**Review by dongsug-song — APPROVED** [review]
approved

**Pull request merged**